templates/aws-stack.yml

```diff
@@ -970,85 +960,43 @@ Resources:
             Version: '2012-10-17'
             Statement:
               - Effect: Allow
-                Action:
-                  - cloudwatch:PutMetricData
+                Action: cloudwatch:PutMetricData
                 Resource: '*'
+        - PolicyName: ReadAgentToken
```

Comment:
I think I want to get rid of this entire resource and either accept a KMS key ID as a parameter of the lambda application for it to generate the correct IAM policy, or adding an `AWS::IAM::Policy` to its generated role here.